### PR TITLE
added .tesselinclude to t2 init explanation

### PR DESCRIPTION
**Pull request by Frijol**
merge once CLI with https://github.com/tessel/t2-cli/pull/639 is released (@rwaldron cc @johnnyman727)

### CODE REVIEW

File: blinky.md [hunk 1]
```diff
@@ -18,13 +18,19 @@ In your command line, make a folder for your Tessel code, then initialize a Tess
 
 `t2 init`
 
-Great! Now you're set up to run code on Tessel. Your "tessel-code" folder now contains a "package.json" with some metadata Node uses for your project, and a file called "index.js".
+Great! Now you're set up to run code on Tessel. Your "tessel-code" folder now contains:
+
+* a "package.json" with some metadata Node uses for your project
+* a ".tesselinclude" file which the T2 CLI uses to optimize file size
```
Comment (johnnyman727) [inline blinky.md]: I would phrase as "optimize bundle size" (since it could be multiple files)

File: blinky.md [hunk 2]
```diff
@@ -18,13 +18,19 @@ In your command line, make a folder for your Tessel code, then initialize a Tess
...
+* a file called "index.js"
```
Comment (johnnyman727) [inline blinky.md]: I would phrase as "an entrypoint file called index.js" because it better described the purpose

**Comment by Frijol** [comment]
@johnnyman727 updated - can you please merge this when you do then next release of the CLI?

**Comment by johnnyman727** [comment]
Yes, if I remember 😟

**Comment by rwaldron** [comment]
Rad

**Pull request merged**